postgresql/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
should this be in this repository ?

Reply:
I'll push new commits to update this PR. I'm testing using `postgresql` bitnami instead of having a local fork.
I'm facing some issues with making it work on OpenShift.

Reply:
ok, no worries, just let me know when I can review.